documentation/API_documentation/QoD_Stable_Latency_API.md

```diff
@@ -0,0 +1,181 @@
+# Overview
+
+<span class="colour" style="color:rgb(0, 0, 0)">The Quality-On-Demand (QoD) API for stable latency provides programmable interface for developers and other users (capabilities consumers) to specify stable data-transfer latency managed by Telco networks without necessity to have in-depth knowledge of the 4G/5G system complexity by abstracting the internal complexity of telecom systems [1].</span>
+
+## 1\. Introduction
+
+There are many real-world use cases which could exploit the potential of programmable networks. Several of these use case implementations are still work in progress or have not been operationalized as developers have no means to configure or request stable latency or bandwidth during congestion from a mobile network operator which is critical. Use cases from industrial (IoT), VR/Gaming, broadcasting, autonomous driving and many others fall in such a category since they demand network communication quality and are sensitive to any change in transmission conditions.
+
+The QoD latency API offers the application developers and users the capability to request for stable latency for a specified App-Flow(s) between application clients and backend services (Application Servers). The developer has a pre-defined set of QoS\_Profiles which he could choose from depending on his latency requirements.
+
+<img src="./resources/QoD_latency_overview.PNG" alt="QoD_LM" title="QoD Latency API Overview">
+
+## 2\. Quick Start
+
+The usage of the stable latency API is based on QoS sessions, which can be created (based on available QoS profiles), queried and deleted.
+The deletion of a requested session can be triggered by the user or can be triggered automatically. The automatic process is triggered either when the user specified duration has reached its limit or default session expiration time has been reached (currently set to 24hrs).
+
+Before starting with the API, the developer needs to provide base-URL, security credentials and App-Flow parameters with specified QoS profile.
+
+**Base-URL**
+The RESTful Stable Throughput API endpoint, for example [**https://telekom-api.developer.telekom.com/5g-latency**](https://telekom-api.developer.telekom.com/5g-throughput)
+
+**Authentication**
+Configure security access keys such as O-Auth2 client credentials to be used by Client applications which will invoke the QoD API.
+
+**QoS Profile**
+Define throughput requirements of the application and identify QoS profile class which maps into the required performance category.
```

Comment:
throughput requirements-> shoudl be latency requirements?

Reply:
Yes, changed in [d93c9fc](https://github.com/camaraproject/QualityOnDemand/pull/18/commits/d93c9fca38d033602f860cad70e14ce2633340c6)